mcu/atmega-hal/src/usart.rs

```diff
@@ -1,3 +1,33 @@
+//! USART
+//!
+//! # Examples
+//!
+//! Complete example source code can be found in the
+//! [repository](https://github.com/Rahix/avr-hal/blob/main/examples/atmega2560/src/bin/atmega2560-usart.rs).
+//!
+//! Terminal input/output using [ufmt](https://crates.io/crates/ufmt/):
```

Comment:
`ufmt` is just used for formatting output.  Maybe leave a note here that `ufmt` is used instead of `core::fmt` because `core::fmt` code quickly grows too large for AVR platforms.

Reply:
Thanks, let me just copy-paste that.